cmd/kubelet/app/server.go

```diff
@@ -502,6 +502,20 @@ func InitializeTLS(kc *componentconfig.KubeletConfiguration) (*server.TLSOptions
 		CertFile: kc.TLSCertFile,
 		KeyFile:  kc.TLSPrivateKeyFile,
 	}
+
+	if ks.TLSCertAuth {
```

Comment:
https://github.com/kayrus/kubernetes/blob/ebbcb11705ce40b3afb3cdcf7542f055444adfb7/cmd/kubelet/app/server.go#L135 is where the nil auth interface is currently set, and where a non nil x509 authenticator could be set based on the client ca option, if specified